### PR TITLE
сhore: update k8s to 1.18

### PR DESCRIPTION
**Pull request by lobkovilya**
### Summary

Based on https://github.com/Kong/kuma/pull/718

- update version of `client-go` 
- update version of minikube and add new target on circleci for k8s 1.18.0
- update `kubectl`

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -443,6 +443,11 @@ jobs:
     - run:
         name: Install Kubectl
         command: make dev/install/kubectl
+    - run:
+        name: Install Conntrack
```
Comment (jakubdyszkiewicz) [inline .circleci/config.yml:447]: What's that? Why we need it? Should we integrate this with our makefiles?

File: Makefile [hunk 2]
```diff
@@ -123,9 +123,9 @@ GOLANGCI_LINT_VERSION := v1.21.0
 
 CI_KUBEBUILDER_VERSION ?= 2.0.0
 CI_KIND_VERSION ?= v0.7.0
-CI_MINIKUBE_VERSION ?= v1.4.0
+CI_MINIKUBE_VERSION ?= v1.9.2
```
Comment (nickolaev) [inline Makefile:126]: I like that :+1:

**Comment by subnetmarco** [comment]
@lobkovilya I agree with @nickolaev's statement. The version we are targeting is 1.18.2 at the moment (the latest `1.18.x`).

File: .circleci/config.yml (hunk 1 above)
Comment (lobkovilya) [reply .circleci/config.yml]: I don't know, for some reason on Ubuntu with `--vm-driver=None` minikube wants `conntrack` in its path (only for kubernetes 1.18.x). We don't have makefile target that setup minikube (like we have for kind) so I'm not sure we need `install/conntrack` as target

Comment (jakubdyszkiewicz) [reply .circleci/config.yml:447]: Ok, it seems we don't need it in makefiles. You can drop a comment here that it is required by minikube with `--vm-driver=None`

**Pull request merged**